workinglog.md

```diff
@@ -12,8 +12,18 @@ In general, the job will make the decision on the data type.
 
 ## Next step
 * Develop a proof of concept for the entire analytics pipeline: extract data into a preferred data format, save the data source into s3, connect athena to it for querying, and finally connect Tableau to athena for visualization.
+  * ✅ Got the code for scraping data as JSON and uploading it to s3.
+  * ❓ I'm not sure how to efficiently do the partition in s3. Do I need to rewrite my code ```webscrape_and_s3upload.py```? The concept of partition sounds very much like creating a bunch of folders for storing data on different days.
```

Comment:
I'm also not exactly sure how we'll want to partition this. We may need to spend time to think this through together for starters.
1. Do we need idempotency? Usually it's good to be able to reproduce, however, we may only be able to scrape random amounts of data based on rate limiting.
2. How often do we run tthis script? That could be a natural partition window, i.e. by day/hour/quarter hour, ettc.
3. that's my understanding of using file partittions (i.e. folders and files to naturally structure which data gets read in at which point. that said, if we are reading with athena?, we may need to understand better how that works.